docs/explanations/how_to_interpret_output_of_fix_plan_api.md

```diff
@@ -0,0 +1,222 @@
+# How to interpret the output of fix plan API endpoint
+
+On the Pro Client version 29, we introduced two distinct endpoints:
+
+* `u.pro.security.fix.cve.plan.v1`
+* `u.pro.security.fix.usn.plan.v1`
+
+Those endpoints can be used to verify the necessary steps for fixing a list of CVEs/USNs.
+When using the CVE endpoint, the expected output is the following one:
+
+```json
+{
+  "_schema_version": "v1",
+  "data": {
+    "attributes": {
+      "cves_data": {
+        "cves": [
+          {
+            "error": null,
+            "expected_status": "EXPECTED_STATUS",
+            "plan": [
+              {
+                "data": {
+                  "binary_packages": [
+                    "package"
+                  ],
+                  "source_packages": [
+                    "package"
+                  ]
+                },
+                "operation": "apt-upgrade",
+                "order": 1
+              }
+            ],
+            "title": "CVE-2017-9233",
+            "warnings": []
+          }
+        ],
+        "expected_status": "EXPECTED_STATUS"
+      }
+    },
+    "meta": {
+      "environment_vars": []
+    },
+    "type": "CVEFixPlan"
+  },
+  "errors": [],
+  "result": "success",
+  "version": "29",
+  "warnings": []
+}
+``` 
+
+From this output, we can see that the **cves_data** object contain two attributes:
+
+* **cves**: A list of CVE objects containing the plan for fixing each of them.
+* **expected_status**: The expected status of the fix operation considering **all** CVEs.
+                       This means that if one CVE cannot be fixed, this field will reflect that.
+
+
+If we take a look at a CVE object, we will see the following structure:
+
+* **title**: The title of the CVE
+* **error**: Any error captured when creating the CVE plan will appear here. The error object
+             will be detailed in a following section.
+* **expected_status**: The expected status of the CVE if the plan was to be executed. There are
+  three possible scenarios: **fixed**, **still-affected** and **not-affected**.
+  The system is considered **still-affected** if there is something that
+  prevents any required packages to be upgraded. Additionally, the system
+  is considered **not-affected**  if the CVE doesn't affect the system at all.
+* **plan**: The necessary plan that would need to be executed to fix the CVE. We will detail each
+  possible plan step on a following section.
+* **warnings**: Warnings that can happen when generating the plan. For example, if the CVE doesn't
+  have a fix available for a given package, this will be reflected here. We will detail the possible
+  warning in a following section.
+
+
+## What operations can appear in the plan array ?
+
+There are four distinct operations that can appear on the plan array. They are:
```

Comment:
```suggestion
There are four distinct operations that can appear in the plan array. They are:
```

Reply:
Also, what is a plan array?

Reply:
Good point, I have added a paragraph explaining what is the idea behind it.
Let me know if it is worth it providing a real case example as well